examples/3d/3d_scene.rs

```diff
@@ -18,14 +18,14 @@ fn setup(
     // circular base
     commands.spawn(PbrBundle {
         mesh: meshes.add(Circle::new(4.0)),
-        material: materials.add(Color::WHITE),
+        material: materials.add(LinearRgba::WHITE),
         transform: Transform::from_rotation(Quat::from_rotation_x(-std::f32::consts::FRAC_PI_2)),
         ..default()
     });
     // cube
     commands.spawn(PbrBundle {
         mesh: meshes.add(Cuboid::new(1.0, 1.0, 1.0)),
-        material: materials.add(Color::srgb_u8(124, 144, 255)),
+        material: materials.add(LinearRgba::rgb(124., 144., 255.)),
```

Comment:
I don't think this u8 -> f32 conversion is correct?

Reply:
In addition, this was an SRGB colour originally, so the proper replacement would be:

```suggestion
        material: materials.add(LinearRgba::from(Srgba::rgb_u8(124, 144, 255))),
```

Which is a little ugly. Makes me wish we had a turbofish `.into::<LinearRgba>()`.